lib/features/local_live_log.js

```diff
@@ -1,9 +1,3 @@
-/**
```

Comment:
I didn't do it because I am not quite intimate about how the module works. Suggestions?